Problem: markdown syntax in tdbe.md caused build failure

> demo-doc-2@1.0.0 docs:build /home/ubuntu/playground/chain-docs
> vuepress build docs

wait Extracting site metadata...
tip Apply theme @vuepress/theme-default ...
Language does not exist: text
Language does not exist: text
Language does not exist: text
Language does not exist: text
(undefined) ./docs/modules/tdbe.md?vue&type=template&id=0fe20faa& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"cb41188e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./docs/modules/tdbe.md?vue&type=template&id=0fe20faa&)
Module build failed (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
SyntaxError: Unexpected token (1:16172)
    at Parser.pp$4.raise (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2757:13)
    at Parser.pp.unexpected (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:647:8)
    at Parser.pp$3.parseExprAtom (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2196:10)
    at Parser.<anonymous> (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:6003:24)
    at Parser.parseExprAtom (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:6129:31)
    at Parser.pp$3.parseExprSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
    at Parser.pp$3.parseExprOps (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1966:19)
    at Parser.pp$3.parseMaybeConditional (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1949:19)
    at Parser.pp$3.parseMaybeAssign (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
    at Parser.pp$3.parseExprList (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2663:20)
    at Parser.pp$3.parseSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2075:29)
    at Parser.pp$3.parseExprSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2050:21)
    at Parser.pp$3.parseMaybeUnary (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
    at Parser.pp$3.parseExprOp (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1985:41)
    at Parser.pp$3.parseExprOps (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1968:91)
 @ ./docs/modules/tdbe.md?vue&type=template&id=0fe20faa& 1:0-488 1:0-488
 @ ./docs/modules/tdbe.md
 @ ./node_modules/@vuepress/core/.temp/internal/page-components.js
 @ ./node_modules/@vuepress/core/lib/client/util.js
 @ ./node_modules/@vuepress/core/lib/client/app.js
 @ ./node_modules/@vuepress/core/lib/client/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/client/clientEntry.js
(undefined) ./docs/modules/tdbe.md?vue&type=template&id=0fe20faa& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/@vuepress/core/node_modules/.cache/vuepress","cacheIdentifier":"520cc585-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??ref--1-1!./node_modules/@vuepress/markdown-loader??ref--1-2!./docs/modules/tdbe.md?vue&type=template&id=0fe20faa&)
Module build failed (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
SyntaxError: Unexpected token (1:16172)
    at Parser.pp$4.raise (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2757:13)
    at Parser.pp.unexpected (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:647:8)
    at Parser.pp$3.parseExprAtom (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2196:10)
    at Parser.<anonymous> (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:6003:24)
    at Parser.parseExprAtom (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:6129:31)
    at Parser.pp$3.parseExprSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
    at Parser.pp$3.parseExprOps (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1966:19)
    at Parser.pp$3.parseMaybeConditional (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1949:19)
    at Parser.pp$3.parseMaybeAssign (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
    at Parser.pp$3.parseExprList (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2663:20)
    at Parser.pp$3.parseSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2075:29)
    at Parser.pp$3.parseExprSubscripts (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2050:21)
    at Parser.pp$3.parseMaybeUnary (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
    at Parser.pp$3.parseExprOp (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1985:41)
    at Parser.pp$3.parseExprOps (/home/ubuntu/playground/chain-docs/node_modules/vue-template-es2015-compiler/buble.js:1968:91)
 @ ./docs/modules/tdbe.md?vue&type=template&id=0fe20faa& 1:0-488 1:0-488
 @ ./docs/modules/tdbe.md
 @ ./node_modules/@vuepress/core/.temp/internal/page-components.js
 @ ./node_modules/@vuepress/core/lib/client/util.js
 @ ./node_modules/@vuepress/core/lib/client/app.js
 @ ./node_modules/@vuepress/core/lib/client/serverEntry.js
 @ multi ./node_modules/@vuepress/core/lib/client/serverEntry.js
Error: Failed to compile with errors.
    at webpack (/home/ubuntu/playground/chain-docs/node_modules/@vuepress/core/lib/node/build/index.js:180:16)
    at finalCallback (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/MultiCompiler.js:254:12)
    at runWithDependencies.err (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/MultiCompiler.js:277:6)
    at done (/home/ubuntu/playground/chain-docs/node_modules/neo-async/async.js:2931:13)
    at runCompilers (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/MultiCompiler.js:181:48)
    at err (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/MultiCompiler.js:188:7)
    at compiler.run (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/MultiCompiler.js:270:7)
    at finalCallback (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/Compiler.js:257:39)
    at hooks.done.callAsync.err (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/Compiler.js:273:13)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/ubuntu/playground/chain-docs/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (/home/ubuntu/playground/chain-docs/node_modules/tapable/lib/Hook.js:154:20)
    at onCompiled (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/Compiler.js:271:21)
    at hooks.afterCompile.callAsync.err (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/Compiler.js:681:15)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/ubuntu/playground/chain-docs/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/home/ubuntu/playground/chain-docs/node_modules/tapable/lib/Hook.js:154:20)
    at compilation.seal.err (/home/ubuntu/playground/chain-docs/node_modules/webpack/lib/Compiler.js:678:31)

npm ERR! Linux 5.3.0-1031-azure
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "docs:build"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! demo-doc-2@1.0.0 docs:build: `vuepress build docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the demo-doc-2@1.0.0 docs:build script 'vuepress build docs'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the demo-doc-2 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     vuepress build docs
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs demo-doc-2
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls demo-doc-2
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/playground/chain-docs/npm-debug.log

--- a/docs/modules/jellyfish-merkle.md
+++ b/docs/modules/jellyfish-merkle.md
@@ -45,7 +45,7 @@ can query the old state, unless we decide to prune the states that is too old.
 
   The hash of the internal node is the root hash of this subtree, which is computed like this:
 
-  ```text
+  ```
   InternalNode[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, A, B, C, D, E, F]
   
     4 ->              +------ root hash ------+
@@ -85,7 +85,7 @@ can query the old state, unless we decide to prune the states that is too old.
 
   For example (notice the path of  `C` is not shortened because it references an internal node):
 
-  ```text
+  ```
   InternalNode[
     0 => Child(leaf),
     3 => Child(leaf),
@@ -187,7 +187,7 @@ fn get(key: H256, version: Version) -> Option<Value> {
 Since the nodes on disk are immutable, so when we need to update a value, we create new nodes. For example, when we
 update `key2`'s value:
 
-```text
+```
                   (on disk)              (in memory)
                     root0                  root1'
                    /     \                /     \
@@ -202,7 +202,7 @@ update `key2`'s value:
 And to optimize batch operations, when a node is already in memory, it's replaced. For example, if we update `key1`'s
 value now,  `root1'` is deleted and replaced by `root1''`:
 
-```text 
+```
                   (on disk)                                     (in memory)
                     root0                                         root1''
                    /     \                                       /     \

--- a/docs/modules/tdbe.md
+++ b/docs/modules/tdbe.md
@@ -158,7 +158,7 @@ TODO: extra rules for CouncilNode -> CommunityNode (needs to unbond / become ina
 internal `Vec<u8>` payloads are expected to be encoded in the TLS standard binary encodings
 (TODO: 
 [the draft MLS architecture doc](https://github.com/mlswg/mls-architecture/blob/master/draft-ietf-mls-architecture.md):
-> In addition, it does not specify a complete wire encoding, but rather a set of abstract data structures which can then be mapped onto a variety of concrete encodings, such as TLS {{?RFC8446}}, CBOR {{?RFC7049}}, and JSON {{?RFC7159}}.
+> In addition, it does not specify a complete wire encoding, but rather a set of abstract data structures which can then be mapped onto a variety of concrete encodings, such as TLS RFC8446, CBOR RFC7049, and JSON RFC7159.
 
 Besides X.509 (to reuse TLS RA stuff) identities in keypackages, we may potentially switch to SCALE to keep it simpler;
 TLS wire format is mainly interesting for test-vectors / unit tests, but the protocol draft is too in flux at the moment.

